acceptance/bundle/templates/lakeflow-pipelines/python/output/my_lakeflow_pipelines/resources/lakeflow_pipelines_etl/lakeflow_pipelines_job.job.yml

```diff
@@ -1,19 +1,21 @@
-# The job that triggers {{template `pipeline_name` .}}.
+# The job that triggers lakeflow_pipelines_etl.
+
 resources:
   jobs:
-    {{template `job_name` .}}:
-      name: {{template `job_name` .}}
+    lakeflow_pipelines_job:
+      name: lakeflow_pipelines_job
 
       trigger:
         # Run this job every day, exactly one day from the last run; see https://docs.databricks.com/api/workspace/jobs/create#trigger
         periodic:
           interval: 1
           unit: DAYS
 
-      email_notifications:
-        on_failure: ${var.notifications}
+      #email_notifications:
```

Comment:
Is the commented code intentional?

Reply:
Yeah for `default-python` I used this approach where there are never emails by default (since they were not effective) + there is no special variable for it (since variables come with a cost)